cdap-ui/app/features/tracker/templates/main.html

```diff
@@ -70,33 +65,33 @@
               </th>
             </thead>
             <tbody>
-              <tr>
-                <td>Name 01</td>
-                <td>-</td>
-                <td>-</td>
+              <tr ng-repeat="metric in MainController.topDatasets.results">
```

Comment:
If this is one time binding lets try to use `<tr ng-repeat="metric in ::MainController.topDatasets.results>` one way binding should be fine here I guess as we load once and forget about it.